extensions/amp-form/0.1/amp-form.js

```diff
@@ -902,14 +909,37 @@ export class AmpForm {
     ).then(request => this.xhr_.fetch(request.xhrUrl, request.fetchOpt));
   }
 
+  /**
+   * Returns the action trust for submit-success and submit-error events.
+   * @param {!ActionTrust} incomingTrust
+   * @return {!ActionTrust}
+   * @private
+   */
+  trustForSubmitResponse_(incomingTrust) {
```

Comment:
What do you think about moving a generic version of this to `action-constants.js`? Degrading trust is 
generally useful beyond recursive form submission, even if that is the target here.

Reply:
I think it's more readable to keep things local until they're actually needed. This avoids the need for readers to jump around and keeps the file directory structure understandable.

Reply:
Hmm, I see what you mean. OTOH if someone is writing similar code elsewhere, I imagine it would be less likely that they are aware this is here and can be repurposed. But I am not too opinionated if you prefer to keep as is.